config.go

```diff
@@ -35,6 +35,7 @@ const (
 	defaultRPCHost            = "localhost"
 	defaultMaxPendingChannels = 1
 	defaultNumChanConfs       = 1
+	defaultTrickleDelay       = 300
```

Comment:
The default should be something much higher than 300 ms. Instead, it should be on the order of 10s of seconds.